File: turn-any-word-into-a-beef-taco/turn-any-word-into-a-beef-taco.py
import re
dictio = {"t":"tomato","l":"lettuce", "c" :"cheese","g":"guacamole","s":"salsa"}
def tacofy(word):
    word = word.lower()
    word = re.sub(r"[^tlcgsaeiou]","",word)
    return ['shell']+["beef" if i in "aeiou" else dictio.get(i) for i in word]+['shell']

File: turn-any-word-into-a-beef-taco/turn-any-word-into-a-beef-taco.js
function tacofy(word) {
  dictio = {"t":"tomato","l":"lettuce", "c" :"cheese","g":"guacamole","s":"salsa"}
  var t = word.toLowerCase().match(/[tlcgsaeiou]/g,"")||[]
  return  ['shell'].concat(t.map(i=>"aeiou".indexOf(i)!=-1 ? "beef":dictio[i])).concat(["shell"])
  
}